bdcs-api.cabal

```diff
@@ -31,7 +31,7 @@ library
     -- other-extensions:
     build-depends:      aeson >= 1.2.2.0 && < 1.3.0.0,
                         base >= 4.9 && < 5.0,
-                        bdcs >= 0.2.1,
+                        bdcs >= 0.2.2,
```

Comment:
There have been a lot of builds of bdcs lately, so you'll need to double check which one you need here.